### PR TITLE
Add artist invite post badges.

### PR DESCRIPTION
**Pull request by alanpeabody**
https://www.pivotaltracker.com/story/show/154422731

### CODE REVIEW

File: src/components/assets/Icons.js [hunk 1]
```diff
@@ -555,3 +555,35 @@ export const TwitterIcon = () =>
...
+          <path d="M8.64615385,11.64 L3.28,11.64 L3.28,17.4876764 L5.96307692,15.5089481 L8.64615385,17.4876764 L8.64615385,11.64 Z" />
+        </g>
+      </g>
+    </g>
+  </SVGIcon>)
+
+
+export const ArtistInviteSubmissionSelectedIcon = () =>
+  (<SVGIcon className="ArtistInviteSubmissionSelectedIcon">
+    <g>
```
Comment (alanpeabody) [inline src/components/assets/Icons.js]: @gblakeman here is the SVG I am having trouble with.

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a linked tracker story for this PR in the description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by gblakeman** [comment]
@alanpeabody Did you want me to merge this or is it still waiting on something?

**Comment by alanpeabody** [comment]
@gblakeman We are waiting on acceptance from @BergerFohr I will follow up with him tomorrow and hopefully get it out.

Thanks for fixing the badge, it looks good!

**Comment by alanpeabody** [comment]
Thanks @gblakeman. It LGTM if we get Lucians approval.

**Pull request merged**